benchclients/python/benchclients/__init__.py

```diff
@@ -1,6 +1,6 @@
 from ._version import __version__
 from .base import BaseClient
 from .conbench import ConbenchClient
-from .logging import log
+from .logging import fatal_and_log, log
 
-__all__ = ["__version__", "BaseClient", "ConbenchClient", "log"]
+__all__ = ["__version__", "BaseClient", "ConbenchClient", "fatal_and_log", "log"]
```

Comment:
I will release `benchclients` to PyPI after this PR is merged, so that `benchalerts` can floor to the version containing these small changes in its `requirements.txt`.